include/cantera/oneD/Flow1D.h

```diff
@@ -670,41 +674,132 @@ class Flow1D : public Domain1D
 
     //! @name Convective spatial derivatives
     //!
-    //! These use upwind differencing, assuming u(z) is negative
+    //! These methods use upwind differencing to calculate spatial derivatives
+    //! for velocity, species mass fractions, and temperature. Upwind differencing
+    //! is a numerical discretization method that considers the direction of the
+    //! flow to improve stability.
     //! @{
+
+    /**
+     * Calculates the spatial derivative of velocity V with respect to z at point j
+     * using upwind differencing.
+     *
+     * For more details on the upwinding scheme, see the
+     * [documentation](../../html/reference/onedim/discretization.html#upwinding).
+     *
+     * @f[
+     *   \frac{\partial V}{\partial z} \bigg|_{j} \approx \frac{V_{j_{\text{loc}}} -
+     *     V_{j_{\text{loc}}-1}}{z_{\text{loc}} - z_{\text{loc}-1}}
```

Comment:
```suggestion
     *   \frac{\partial V}{\partial z} \bigg|_{j} \approx \frac{V_{\text{loc}} -
     *     V_{\text{loc}-1}}{z_{\text{loc}} - z_{\text{loc}-1}}
```
(or you could switch to `\ell` as in the other docs.